Widgets/widget-ActiveIncidentsBySeverity.json

```diff
@@ -1,5 +1,5 @@
 {
-  "id": "6b25afb6-7a95-4878-a24c-24cf067eb38a",
+  "id": "active-incidents-by-severity",
```

Comment:
The dashboard is still using the old ID.

![image](https://user-images.githubusercontent.com/5429033/35193869-324d785e-feb2-11e7-81af-5adf19c9f3ea.png)

Reply:
The widgets in the dashboard are copies of the widgets when we created them, they don't update if we update the widgets.